CHANGELOG.md

```diff
@@ -8,7 +8,7 @@ and this project adheres to [Early Semantic Versioning](https://docs.scala-lang.
 
 ### Changed
 
-* The project was migrated to the [Typelevel](https://github.com/typelevel) organisation. The artifact is now published under the `org.typelevel` group ID.
+* The project was migrated to the [Typelevel](https://github.com/typelevel) organisation. The artifact is now published under the `org.typelevel` group ID. It remains binary compatible with the 0.3.x series.
```

Comment:
Actually, not really. You should change a publishing org without changing a package name, because otherwise eviction doesn't work*. But as soon as you change the package name, it's no longer bincompatible.

\* sadly any org/package change for sbt-plugins is a mess, recently went through this in:
- https://github.com/sbt/sbt-git/issues/216

Reply:
Hmm I didn't even think about sbt eviction, that's even more difficult to resolve.